rental.rb

```diff
@@ -0,0 +1,12 @@
+class Rental
+  attr_accessor :date
+  attr_reader :book, person
+
```

Comment:
- [ ]  You should have Getters and setters for @Rentals attribute as well. this will give @rental both write and read methods.